tests/misc/projects/Issue8534/compile-fail.hxml.stderr

```diff
@@ -0,0 +1 @@
+Main.hx:3: characters 3-36 : Array<Null<Int>> has no field joint (Suggestion: join)
```

Comment:
Didn't check out the PR branch, but 3-36 doesn't seem like the correct range?

Reply:
`'["1", "2", "3"].map(Std.parseInt)'.length == 33` so it seems correct to me

Reply:
My idea was to highlight only `joint`, not the other way around.

Reply:
My bad, corrected :D